### DISCUSSION TITLE
BT-01(c)-Procedure should be mandatory if BT-01(d)-Procedure was filled

### DISCUSSION THREAD

**Discussion by magdalenasPwC** [OP-TED/eForms-SDK#997] [Q&A] [answered]
We encountered an issue while sending notice type 16. Field BT-01(d)-Procedure was filled, BT-01(c)-Procedure left empty. Field validation did not show any mistakes. Error occurred in publication response that xml is not correct. The Xml lacked ID which comes from BT-01(c)-Procedure field. Therefore BT-01(c)-Procedure field should be mandatory if BT-01(d)-Procedure was filled

**Comment by YvesJo** [comment, marked as answer]
Hi,
There are various cases that use the same aggregate element "cac:ProcurementLegislationDocumentReference" and the basic element "cbc:ID" is carrying the semantic that allows to differentiate the case. In absence of cbc:ID information, it is impossible to know whether the description is about "Cross Border Law", "Other Legislation with ID", or "Other Legislation without ID".
Therefore having only "cbc:DocumentDescription" is not sufficient to identify what is actually missing (BT-09(a)-Procedure, BT-01(e)-Procedure or BT-01(c)-Procedure). This has to be addressed on the UI side to ensure the correct cbc:ID will be specified and prevent the XSD non-conformance.
